.github/workflows/haskell.yml

```diff
@@ -93,10 +93,10 @@ jobs:
         cache-name: cache-cabal-build-130824
       with:
         path: ${{ steps.setup-haskell.outputs.cabal-store }}
-        key: ${{ runner.os }}-${{ matrix.ghc }}-${{ env.cache-name }}-${{ hashFiles('**/*.cabal') }}-${{ hashFiles('**/cabal.project') }}-${{ hashFiles('**/cabal.project.debug') }}
+        key: ${{ runner.os }}-${{ matrix.ghc }}-${{ env.cache-name }}-${{ hashFiles('*.cabal') }}-${{ hashFiles('cabal.project*') }}-${{ matrix.cabal-project-file }}
```

Comment:
So this includes the `cabal.project` that is just a symlink? Does it matter, or do we consistently use `matrix.cabal-project-file` to pick the one to use?

Reply:
[`hashFiles`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#hashfiles) requires a file pattern that allows globs. Presumably the glob results in the same files as listed here:

```
❯ echo cabal.project*
cabal.project cabal.project.blockio-uring cabal.project.debug cabal.project.local cabal.project.release
```

Just to err on the safe side, I thought it would be good to hash all configuration files, i.e., `*.cabal` and `cabal.project.*`. Omitting or including `cabal.project` in this fine would have both been fine, because we use `matrix.cabal-project-file` consistely to pick the actual configuration for builds/tests

Reply:
Okay, then it seems good. Was just wondering whether `cabal.project.*` would be better.